src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -36,8 +32,8 @@ <h4>{{ data.name }} </h4>
     [responsive]="true"
     [resizable]="false"
     [draggable]="false"
-    [closable]="isEditMode()"> 
```

Comment:
Replace this dialog with `dot-dialog` and in every code you touch if you see an standalone `p-dialog` please replace it with a `dot-dialog`